civis/tests/test_utils.py

```diff
@@ -294,7 +299,7 @@ def test_no_retry_on_connection_error(mock_session):
 def test_retry_respect_retry_after_headers(mock_session):
     expected_call_count = 0
     max_calls = 3
-    retry_after = 3
+    retry_after = 1
```

Comment:
Not sure if this change was behind the intent of the ticket but by reducing this wait time the test runs much more quickly (this is the longest running test currently).

Reply:
Seems fine to me.

Please feel free to tag me again when this PR is ready for review.

Reply:
Trying to shorten a couple more